tox.ini

```diff
@@ -38,6 +47,12 @@ deps =
 [testenv:py34]
 basepython=python3.4
 
+[testenv:py34-with-django]
+basepython=python3.4
+deps =
+    -r{toxinidir}/requirements-dev.txt
+    django
```

Comment:
Would you mind terribly if if was `django>=1.9.0,<1.11.0`? I'd like to use this with a 1.9 project